src/Aggregate/AggregateRepository.php

```diff
@@ -0,0 +1,279 @@
+<?php
+/**
+ * This file is part of the prooph/event-sourcing.
+ * (c) 2014-2016 prooph software GmbH <contact@prooph.de>
+ * (c) 2015-2016 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventSourcing\Aggregate;
+
+use ArrayIterator;
+use Prooph\Common\Messaging\Message;
+use Prooph\EventSourcing\Snapshot\SnapshotStore;
+use Prooph\EventStore\EventStore;
+use Prooph\EventStore\Stream\Stream;
+use Prooph\EventStore\Stream\StreamName;
+
+/**
+ * Class AggregateRepository
+ *
+ * @package Prooph\EventSourcing\Aggregate
```

Comment:
what's the point of duplicating the name and the namespace in the class comment? It doesn't add any value and it forces contributors to maintain these comments over time.